console/src/main/scripts/plugins/metrics/html/directives/explorer/resource-tree.html

```diff
@@ -0,0 +1,6 @@
+<div class="hk-resource-tree">
+  <label>Resource children tree</label>
+  <div ng-if="$ctrl.childrenTree.length !== 0">
+    <hk-boot-nav-tree tree-data="$ctrl.childrenTree" on-node-selected="$ctrl.onLoadMetrics({resource: resource})"></hk-boot-nav-tree>
```

Comment:
The controllerAs label `$ctrl` is not very descriptive (they are all controllers) and the `$` makes it feel like its a special Angular variable that shouldn't be touched.